pkgs/applications/networking/instant-messengers/chatterino2/default.nix

```
@@ -1,28 +1,42 @@
-{ stdenv, pkgconfig, fetchFromGitHub, qtbase, qtsvg, qtmultimedia, qmake, boost, openssl }:
+{ stdenv, pkgconfig, fetchFromGitHub, qtbase, qtsvg, qtmultimedia, qmake, boost, openssl, wrapQtAppsHook }:
 
 stdenv.mkDerivation rec {
   pname = "chatterino2";
-  version = "unstable-2019-05-11";
+  version = "v2.1.7";
+
   src = fetchFromGitHub {
-    owner = "fourtf";
+    owner = "Chatterino";
     repo = pname;
-    rev = "8c46cbf571dc8fd77287bf3186445ff52b1d1aaf";
-    sha256 = "0i2385hamhd9i7jdy906cfrd81cybw524j92l87c8pzrkxphignk";
+    rev = version;
```

```suggestion
    rev = "$v${version};
```

please no letters in `version`.